app.js

```diff
@@ -0,0 +1,182 @@
+'use strict';
+
+// product objects go into the timeOut array when they're shown, and held there until their canDisplay counter goes back to zero. I think it's supposed to be after 1 click, but I was having issues setting the number lower than 2.
+var timeOut = [];
+// all of the product objects go into the imgArr when they can be displayed.
+var imgArr = [];
+// declare variables for the elements in index.html where I want the pictures to show.
+var firstImage = document.getElementById('first-image');
+var secondImage = document.getElementById('second-image');
+var thirdImage = document.getElementById('third-image');
+
+// this is where the number of times each image is shown and clicked will be displayed.
+var results = document.getElementById('results-field');
+
+// I have these as global variables for now because they store the current three images and I need to use them in multiple functions. I'll probably refactor this to be more efficient.
+var first;
+var second;
+var third;
+
+// variable to store the total number of clicks. I need to know when the user gets to 25.
+var counter = 0;
+
+// object constructor for the products to be displayed, with properties to track how many times shown, clicks, and the file path to the corresponding image.
+function Product(name, path, shown, clicks, canDisplay) {
```

Comment:
you don't need to pass `shown`, `clicks`, `canDisplay` into your constructor. you can set them as 0 in the constructor and then you don't have to pass in all those zeros.